docs/guidance/map/framework_agreements.md

```diff
@@ -85,44 +85,45 @@ The following guidance describes how to model the different stages of a framewor
 
 ### Invitation to participate in the first stage of a framework agreement procedure
 
-* Create a release and add 'tender' to the `tag` array
+* Create a release and add 'tender' to the `tag` array.
 * For each buyer that will participate in the framework agreement, add an `Organization` object to the `parties` array, add 'buyer' to its `.roles`, and populate its other fields.
   * If all potential buyers can participate, don't add them to the `parties` array. Instead, add a note in the [publication policy](../publish) to explain how users should interpret a contracting process with no buyers.
   * If a single buyer will participate in the framework agreement, set the `buyer.id` and `buyer.name` to match the buyer's object in the `parties` array. Otherwise, don't populate `buyer`.
 * In the `tender` section, set:
+  * `tender.procurementMethod` in line with the competitive conditions of the first stage of the framework agreement. Since framework agreements usually involve qualification, this is most often 'selective'.
```

Comment:
```suggestion
  * `tender.procurementMethod` according to the competitive conditions of the first stage of the framework agreement. Since framework agreements typically involve qualification, this is most often 'selective'.
```